src/Features/Core/Portable/Completion/Providers/AbstractKeywordCompletionProvider.cs

```diff
@@ -60,24 +60,24 @@ public int GetHashCode(RecommendedKeyword obj)
                     cancellationToken).ConfigureAwait(false);
 
                 var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
-                var filterSpan = this.GetTextChangeSpan(text, position);
 
                 foreach (var keyword in keywords)
                 {
-                    context.AddItem(CreateItem(keyword, filterSpan));
+                    context.AddItem(CreateItem(keyword, context.DefaultItemSpan));
                 }
             }
         }
 
-        protected virtual CompletionItem CreateItem(RecommendedKeyword keyword, TextSpan filterSpan)
+        private static ImmutableArray<string> s_Tags = ImmutableArray.Create(CompletionTags.Intrinsic);
+
+        protected virtual CompletionItem CreateItem(RecommendedKeyword keyword, TextSpan span)
         {
-            return new KeywordCompletionItem(
-                this,
+            return CommonCompletionItem.Create(
                 displayText: keyword.Keyword,
-                filterSpan: filterSpan,
-                descriptionFactory: c => Task.FromResult(keyword.DescriptionFactory(c)),
+                span: span,
+                description: keyword.DescriptionFactory(CancellationToken.None),
```

Comment:
Are we now synchronously computing descriptions?  That seems bad.  It was highly necessary to avoid doing this in practice for perf reasons.